mlflow/store/rest_store.py

```diff
@@ -191,7 +191,7 @@ def set_tag(self, run_id, tag):
 
     def delete_tag(self, run_id, key):
         """
-        Delete a tag for the specified run with the specified tag key.
+        Delete a tag for the specified run with the specified tag key. This is irreversible.
         :param run_id: String id for the run
         :param key: name of the tag to delete
```

Comment:
String id for the run -> String ID of the run
name of the tag to delete -> Name of the tag to delete